components/contact/ContactForm2.vue

```diff
@@ -0,0 +1,813 @@
+<template>
+  <div class="contactForm font-body mt-8 ml-12">
+    <div>
+      <div v-if="didAttemptSubmit && invalidFields.length" class="error-list">
+        <h2 ref="displayErrors" class="text-xl ml-2 text-red-600">
+          {{ $t('contactValidation.errorListTitle') }}
+        </h2>
+        <ul
+          class="list-disc text-sm text-red-600 ml-2 italic"
+          style="list-style-position: inside"
+        >
+          <li v-for="invalidField in invalidFields" :key="invalidField">
+            {{ $t('contact.' + invalidField) }}
+          </li>
+        </ul>
+      </div>
+      <h2 class="title font-display">
+        {{ $t('contact.edit') }}
+      </h2>
+
+      <div>
+        <p class="orange font-body">
+          {{ $t('contact.type') }}
+        </p>
+      </div>
+
+      <div class="flex mb-4">
+        <div class="w-5/12 margins">
+          <select
+            v-model="contactInfo.type"
+            class="formSelect"
+            @change="onStanding($event)"
+          >
```

Comment:
Still seems to be an issue here. This drop down does not populate. Is it possibly a CSS error?

Reply:
DB data must match the values preset for the app. or wont work. 
try adding a contact using the app